utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+// Adapted from https://github.com/prometheus-community/prom-label-proxy/blob/02d43edb82b7d139f4a4e41912ad903bff46d5c4/injectproxy/utils.go#L22
+func PrometheusAPIError(w http.ResponseWriter, errorMessage string, code int) {
+	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	w.WriteHeader(code)
+
+	res := map[string]string{"status": "error", "errorType": "observatorium-api", "error": errorMessage}
```

Comment:
I wonder if we should establish in the API spec what error types there can be, whether there should always be just one etc., but either way we should have this documented in the spec. For starters, this is fine for me though :+1:.